samples/wfi32iot/basic-sample/src/iotconnect_app.c

```diff
@@ -195,7 +182,7 @@ static void on_connection_status(IotConnectConnectionStatus status) {
     }
 }
 
-static void publish_telemetry() {
+static void publish_telemetry(int warming_up, int connections[]) {
```

Comment:
Do we really need this variable? Can't we just record boot time in a local static variable and substract current time from it. It burdens the application to track this. Also the name warming_up should probably be renamed to warmup_secs or warmup_seconds, seconds_since_start or something like that.